lib/model/request/auth_token_request.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:core';
+
+import 'package:flutter_survey/env.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'auth_token_request.g.dart';
+
+const _grantTypePassword = "password";
+
+@JsonSerializable()
+class AuthTokenRequest {
+  final String email;
+  final String password;
+  String grantType = _grantTypePassword;
```

Comment:
Can we declare a `final` here? 🤔

Reply:
@markgravity done in https://github.com/nimblehq/ic-flutter-avishek/pull/34/commits/729dcb2257aa3b76d4e551fd5af4f7064ac98ed3